include/geos/ggraph.h

```diff
@@ -172,5 +176,4 @@ typedef void graphicStr;
 /* ESC_PUTSTRING can't be implemented - it needs text, not pointer to it
    #define ESC_PUTSTRING(x,y,text) (char)6, (unsigned)(x), (char)(y), (text), (char)NULL
 */
-
```

Comment:
Sorry for the nitpicking, but all header files have a blank line before the multi inclusion guard close. Please don't remove it here.

Reply:
Sorry, not sure why that happened.  Should be fixed now.  No problem nitpicking - its the only way Ill learn how you guys want this.  I welcome it.